bin/fin

```diff
@@ -151,6 +170,19 @@ wsl_path ()
 	echo $(wslpath "$@" | tr -d '\r')
 }
 
+# Convert version string like 1.2.3.4 to integer for comparison
+# param $1 version string of 4 components max (e.g., 1.10.3.0)
+ver_to_int ()
+{
+	echo "$@" | awk -F. '{ printf("%d%03d%03d%03d", $1,$2,$3,$4); }'
+}
+
+# docker-compose v1.x.x vs docker compose v2.x.x
```

Comment:
I would advocate to only support dc v 2.x and get rid of everything 1.x related

Reply:
Yeah, keeping v1 made sense during v2-beta for testing and comparison. With the v2 release and all tests passing, we can cleanup the v1 stuff.

Reply:
I think i left similar comments that if we are forcing the v2 on the user we could remove any conditionals around checking for v1

Reply:
Removed all references and support for Docker Compose v1